src/_reducers/feedback.reducer.js

```diff
@@ -3,24 +3,24 @@ import constants from '../_constants';
 const { LOAD_FEEDBACK_SUCCESS, LOAD_FEEDBACK_FAILURE } = constants;
 
 const initialState = {
-    feedback: [],
-    feedbackCount: 0,
-}
+  feedback: [],
+  feedbackCount: 0,
```

Comment:
Is there a reason you are storing the `feedbackCount` in the reducer? Is this something 

1. used in pagination,
2. required by the backend
3. returned by the backend

Let me know

Reply:
It was already added before I raised this PR. I can remove it if it's not needed.

Reply:
@andela-fojuri you still haven't answered the questions

Reply:
I'm not sure of the main reason the person that worked on this was storing the feedbackCount in the reducer. It's being used in pagination but not returned/required by the backend.